src/colmap/controllers/feature_extraction.cc

```diff
@@ -279,6 +279,11 @@ class FeatureWriterThread : public Thread {
             image_data.camera.has_prior_focal_length ? " (Prior)" : "");
         LOG(INFO) << StringPrintf("  Features:        %d",
                                   image_data.keypoints.size());
+        if (image_data.mask.Data()) {
+          LOG(INFO) << "  Mask Used:       Yes";
+        } else {
+          LOG(INFO) << "  Mask Used:       No";
+        }
```

Comment:
I would guess that 99% of use cases involve no masks. I suggest, let's not clutter the logs in this case.
```suggestion
        if (image_data.mask.Data()) {
          LOG(INFO) << "  Mask:            Yes";
        }
```

Reply:
That's a fair assumption to make!